modules/ROOT/pages/provisioning-openstack.adoc

```diff
@@ -0,0 +1,97 @@
+= Provisioning Fedora CoreOS on OpenStack
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on an
+OpenStack cloud environment, either private, or public (like https://vexxhost.com/[VEXXHOST]).
+
+== Prerequisites
+
+Before provisioning a FCOS machine, you must have an Ignition configuration file
+containing your customizations. If you do not have one, see
+xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to an OpenStack environment and a functioning
+https://docs.openstack.org/python-designateclient/latest/user/shell-v2.htm[`openstack` CLI]. 
+Typically you'll https://docs.openstack.org/python-openstackclient/latest/configuration/index.html[configure the client]
+by using a `clouds.yaml` file or via environment variables. If you're starting from scratch, this
+environment may need networks, ssh keypairs, security groups, etc.. set up. Please consult the
+https://docs.openstack.org/[OpenStack Documentation] to learn more.
+
+== Downloading an OpenStack Image
+
+Fedora CoreOS is designed to be updated automatically, with different schedules per stream.
+Once you have picked the relevant stream, download, verify, and decompress the latest
+OpenStack image:
+
+[source, bash]
+----
+STREAM='stable'
+coreos-installer download --decompress -s $STREAM -p openstack -f qcow2.xz
+----
+
+Alternatively, you can manually download an OpenStack image from the
+https://getfedora.org/coreos/download?tab=cloud_operators[download page].
+Verify the download, following the instructions on that page, and decompress it.
+
+== Uploading the Image to OpenStack
+
+.Create the FCOS image in OpenStack
+[source, bash]
+----
+FCOS_VERSION='...'
+FILE=fedora-coreos-${FCOS_VERSION}-openstack.x86_64.qcow2
```

Comment:
Minor/optional: meh... I'd just pick a sample version here and roll with that. I think it's understood that they'd sub in the filename they actually have.

Reply:
Yeah, I copy and pasted this from another guide. I think all of them could probably use a single person going through and updating each of them so the language and common steps are a bit more consistent.